.github/workflows/sui-ccip-test.yml

```diff
@@ -61,6 +61,22 @@ jobs:
           --health-retries=5
 
     steps:
+      # 0) Free up disk space
```

Comment:
remove some pre-installed apps on the runner to avoid `tee: /tmp/gotest.log: No space left on device` issue